.github/workflows/performance-tests-job-configs/jDBC.txt

```diff
@@ -0,0 +1,30 @@
+###############################################################################
```

Comment:
Please remove these multiple symbols, there is no need to section the license header.